server/services/game_service.go

```diff
@@ -267,3 +307,24 @@ func createBoard(gameState *pogo.GameState) []pogo.BoardSpace {
 	}
 	return boardSpaces
 }
+
+func logGameTurn(game *GameInstance, inputs []pogo.GameInputMsg) {
```

Comment:
This function will log out the current state and the given inputs so that we can input that data into our unit tests if something goes wrong.